src/app/routes/api/v3/admin/workspaces/workspaces.routes.ts

```diff
@@ -0,0 +1,70 @@
+import { Router } from 'express'
+
+import {
+  logAdminAction,
+  withUserAuthentication,
+} from '../../../../../modules/auth/auth.middlewares'
+import * as WorkspaceController from '../../../../../modules/workspace/workspace.controller'
+
+import { WorkspacesFormRouter } from './workspaces.form.routes'
+
+export const WorkspacesRouter = Router()
+WorkspacesRouter.use(withUserAuthentication)
+WorkspacesRouter.use(logAdminAction)
+
+WorkspacesRouter.use(
+  '/:workspaceId([a-fA-F0-9]{24})/forms',
+  WorkspacesFormRouter,
+)
+
+WorkspacesRouter.route('/')
+  /**
+   * List the workspaces that the user owns
+   * @security session
+   *
+   * @returns 200 with a list of workspaces owned by the user
+   * @returns 401 when user is not logged in
+   * @returns 422 when user of given id cannnot be found in the database
+   * @returns 500 when database errors occur
+   */
+  .get(WorkspaceController.getWorkspaces)
+
+  /**
+   * Create a new workspace
+   * @security session
+   *
+   * @returns 200 with newly created workspace
+   * @returns 400 when Joi validation fails
+   * @returns 401 when user does not exist in session
+   * @returns 409 when a database conflict error occurs
+   * @returns 422 when user of given id cannnot be found in the database
+   * @returns 500 when database error occurs
+   */
+  .post(WorkspaceController.createWorkspace)
+
+WorkspacesRouter.route('/:workspaceId([a-fA-F0-9]{24})')
+  /**
+   * Delete an existing workspace
```

Comment:
can we more clearly differentiate between the different kinds of deletes in the comments? not clear whether this is deleting workspace-only, workspace + forms or multiple forms